commands/src/main/java/org/wildfly/extras/creaper/commands/orb/AuthValues.java

```diff
@@ -0,0 +1,22 @@
+package org.wildfly.extras.creaper.commands.orb;
+
+public enum AuthValues {
+    NONE("None"),
+    SERVER_AUTH("ServerAuth"),
+    CLIENT_AUTH("ClientAuth"),
+    MUTUAL_AUTH("MutualAuth");
+
+    private final String value;
+
+    AuthValues(String auth) {
+        this.value = auth;
+    }
+
+    public String toString() {
+        return value;
+    }
+
+    public String getValue() {
+        return value;
+    }
```

Comment:
ah, sure. I had to be unconscious when fixing the issue last time.

...don't worry master, I won't disappoint you again :]